src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -1,5 +1,71 @@
 @import 'extra-actions';
 
+.ez-btn {
+    display: inline-block;
+    padding: calculateRem(7px) calculateRem(24px);
+    outline: none;
+    border: none;
+    border-radius: $border-radius;
+    text-decoration: none;
+    font-size: calculateRem(13px);
+    transition: all 0.2s $ez-admin-transition;
+
+    &:hover,
+    &:focus,
+    &:active {
+        border: none;
+        outline: none;
+        text-decoration: none;
+    }
+
+    &--primary {
+        color: $ibexa-white;
+        background-color: $ibexa-color-primary;
+
+        &:hover {
+            color: $ibexa-white;
+            background-color: $ibexa-color-primary-light;
+        }
+
+        &:active {
+            background-color: $ibexa-color-primary-light;
+            box-shadow:inset 0 0 0 calculateRem(2px) $ibexa-color-primary;
+        }
+    }
+
+    &--secondary {
+        color: $ibexa-color-primary;
+        background-color: transparent;
+        box-shadow:inset 0 0 0 calculateRem(1px) $ibexa-color-primary;
+
+        &:hover {
+            color: $ibexa-color-primary;
+            background-color: $ibexa-color-primary-pale;
+        }
+
+        &:active {
+            background-color:$ibexa-color-primary-pale;
+            box-shadow:inset 0 0 0 calculateRem(2px) $ibexa-color-primary
+        }
+    }
+
+    &--no-border {
+        color: $ibexa-color-primary;
+    }
+
+    &--login {
```

Comment:
not sure if `--login` modifier should be in this file, it seems quite specific use, @dew326  ?

Reply:
But this is button